gammapy/background/off_data_background_maker.py

```diff
@@ -230,7 +230,46 @@ def save_models(self, modeltype):
             Type of the background modelisation
         """
         for ngroup in range(self.ntot_group):
-            self.save_model(modeltype, ngroup)
+            self.save_model(modeltype, ngroup, smooth)
+
+    def smooth_models(self, modeltype):
+        """Smooth the bkg model for each group
+
+        Parameters
+        ----------
+        modeltype : {'3D', '2D'}
+            Type of the background modelisation
+
+        Returns
+        -------
+
+        """
+        for ngroup in range(self.ntot_group):
+            self.smooth_model(modeltype, ngroup)
+
+    def smooth_model(self, modeltype, ngroup):
+        """Smooth the nkg model for one group
+
+        Parameters
+        ----------
+        modeltype : {'3D', '2D'}
+            Type of the background modelisation
+        ngroup : int
+            Groups ID
+        Returns
+        -------
+
+        """
+        if modeltype == "3D":
+            if str(ngroup) in self.models3D.keys():
+                self.models3D[str(ngroup)].smooth()
+            else:
+                log.info("No run in the band {}".format(ngroup))
```

Comment:
"band" -> "group"
(we can discuss re-naming in the future, but for now let's be consistent and use "group" everywhere.